util.go

```diff
@@ -0,0 +1,8 @@
+package chat
```

Comment:
This only kind of applies since it's the filename. But, in general, avoid "util" packages. Might name this type of file `log.go`, which makes it clear there is logging logic in it.

https://blog.golang.org/package-names

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common